pkgs/servers/nextcloud/packages/24.json

```diff
@@ -110,9 +110,9 @@
     ]
   },
   "news": {
-    "sha256": "17kz5499jkv43w8wcd1p982hpkw2akgzpv9cjj8qqjhvzv4qr171",
-    "url": "https://github.com/nextcloud/news/releases/download/21.0.0-beta1/news.tar.gz",
-    "version": "21.0.0-beta1",
+    "sha256": "1j7iqnjm1f1kqhhfw1kc36a740mnlkwf4yn2rszlkdgizmfcwvra",
+    "url": "https://github.com/nextcloud/news/releases/download/21.0.0/news.tar.gz",
+    "version": "21.0.0",
```

Comment:
this is a downgrade

Reply:
No, 21.0.0 comes after the beta: https://github.com/nextcloud/news/releases